README.md

```diff
@@ -415,7 +416,8 @@ to set `true`; it's effectively a shortcut for `foo=true`).
   overhead (compression will be slower).
 
 - `drop_console` -- default `false`.  Pass `true` to discard calls to
-  `console.*` functions.
+  `console.*` functions. If you want to discard say `console.info` only
+  and/or retain parameters with side effects, use `pure_funcs` instead.
```

Comment:
If you wish to drop a specific function call such as `console.info` and/or retain side effects from function arguments after dropping the function call then use `pure_funcs` instead.